kmp-nativecoroutines-compiler/build.gradle.kts

```diff
@@ -34,13 +34,15 @@ java {
 }
 
 tasks.compileKotlin.configure {
-    kotlinOptions {
-        freeCompilerArgs = listOf("-Xjvm-default=all")
+    compilerOptions {
+        freeCompilerArgs.add("-Xjvm-default=all")
     }
 }
 
 tasks.test {
-    systemProperty("kotlin.internal.native.test.nativeHome", NativeCompilerDownloader(project).compilerDirectory.absolutePath)
+    val compilerDownloader = NativeCompilerDownloader(project)
+    compilerDownloader.downloadIfNeeded()
```

Comment:
This isn't ideal need to find a way to depend on the new download logic

Reply:
https://youtrack.jetbrains.com/issue/KT-66929/Allow-Gradle-tasks-to-depend-on-native-compiler